packages/api-request-builder/test/create-service.spec.js

```diff
@@ -66,7 +73,7 @@ describe('createService', () => {
     )
   })
 
-  it('should throw if definition prop (features) is a non empty array', () => {
+  it('should throw if definition prop (features) is an empty array', () => {
```

Comment:
In that case the error message is wrong: 
>Definition requires `features` to be a non empty array

Should instead be: 
>Definition requires `features` to be an empty array

...and [here](https://github.com/commercetools/nodejs/blob/43-projectKey-main-option/packages/api-request-builder/test/create-service.spec.js#L83), you still expect it to throw when features is **empty** 😏